content/configuration/source.md

```diff
@@ -0,0 +1,333 @@
+---
+menutitle: "Source"
+title: "Source"
+weight: 23
+---
+
+The source settings contains:
+- Resource list you want to watch,
+- Namespaces you want to filter,
+- The type of events you want to get notifications about,
+- Settings to monitor updates for the specific resource fields,
+- Ability to turn on/off specific recommendations per a given source.
+
+Sources are bound to specific channels in the communications configuration. To learn more, read the [Source and Executor Bindings](/configuration/communication/#source-and-executor-bindings) section.
+
+## Recommendations
+
+For every source you can configure recommendations related to Kubernetes resources. The full recommendation configuration is as follows:
+
+```yaml
+kubernetes:
+
+  # ... trimmed ...
+
+  recommendations:
+    # Recommendations for Pod Kubernetes resource.
+    pod:
+      # If true, notifies about Pod containers that use `latest` tag for images.
+      noLatestImageTag: true
+      # If true, notifies about Pod resources created without labels.
+      labelsSet: true
+    # Recommendations for Ingress Kubernetes resource.
+    ingress:
+      # If true, notifies about Ingress resources with invalid backend service reference.
+      backendServiceValid: true
+      # If true, notifies about Ingress resources with invalid TLS secret reference.
+      tlsSecretValid: true
+```
+
+### Merging strategy
+
+If multiple source bindings are specified for a given communication channel, the recommendations are merged. If any bound source has a given recommendation enabled, it is enabled for the channel.
```

Comment:
In general, it LGTM but we will need to think about common approach for merging. Currently, each "type" is merged in a different way 😄 
1. Source.Kubernetes.Resources - append
2. Source.Kubernetes.Recommendations - the enabled wins (binding order doesn't matter)
3. Executor.Kubectl.Commands.Verbs - append
4. Executor.Kubectl.Commands.Resources  - append
5. Executor.Kubectl.Commands.RestrictAccess - override (binding order is important)

IMO it would be nice if 2nd and 5th would work in the same way.

Reply:
🤔 Definitely to discuss later 👍